lang/da.js

```diff
@@ -9,23 +9,23 @@ export default {
 	"components.dialog.close": "Luk denne dialogboks",
 	"components.form-element.defaultError": "{label} er ugyldigt.",
 	"components.form-element.defaultFieldLabel": "Felt",
-	"components.form-element.input.email.typeMismatch": "",
-	"components.form-element.input.number.rangeOverflow": "",
-	"components.form-element.input.number.rangeUnderflow": "",
-	"components.form-element.input.text.tooShort": "",
-	"components.form-element.input.url.typeMismatch": "",
+	"components.form-element.input.email.typeMismatch": "E-mail er ikke gyldig",
+	"components.form-element.input.number.rangeOverflow": "Tal skal være lavere end {max}",
+	"components.form-element.input.number.rangeUnderflow": "Tal skal være højere end {min}",
+	"components.form-element.input.text.tooShort": "{label} skal være på mindst {minlength} tegn",
+	"components.form-element.input.url.typeMismatch": "URL-adresse er ikke gyldig",
 	"components.form-element.valueMissing": "{label} er påkrævet.",
-	"components.form-error-summary.errorSummary": "",
+	"components.form-error-summary.errorSummary": "Der blev fundet {count, plural, one { was 1 error } other { were { count } errors }} i den information, du sendte",
```

Comment:
Translators missed some English here 😞

Reply:
Yeah :( Should I message Jeff about that?

Reply:
Or you can use the slack workflow (in the #localization slack channel, click the lightning bolt and fill out the "Translation Error Report").  I _think_ that will flag it for them to re-request it, but you could check with Jeff to confirm